.eslint-dictionary.json

```diff
@@ -1,201 +1,394 @@
 [
+  "accessor",
+  "adaptor",
+  "adaptors",
+  "addgraphqldatasource",
+  "admingroup",
   "adminui",
+  "aggs",
+  "allowlist",
+  "alphanumerics",
   "amazonaws",
+  "amazoncognito",
+  "amd64",
+  "amplifyadmin",
```

Comment:
I think this should be an error

Reply:
There is a url and string literal referencing this word.